app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    c_list = command.split()
+    if len(c_list) != 3 or c_list[0] != "cp":
+        raise ValueError("Not correct command")
```

Comment:
The error message 'Not correct command' could be more descriptive. Consider specifying what part of the command is incorrect.